articles/openshift/oshift-ref-cluster-monitoring.md

```diff
@@ -0,0 +1,53 @@
+---
+title: OpenShift integrated cluster monitoring | UKCloud Ltd
+description: Provides information on the Prometheus monitoring stack deployed with OpenShift v3.11
+services: openshift
+author: Kieran O'Neill
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift integrated cluster monitoring
+toc_fullpath: Reference/oshift-ref-cluster-monitoring.md
+toc_mdlink: oshift-ref-cluster-monitoring.md
+---
+
+# OpenShift integrated cluster monitoring
+
+### Overview
+
+OpenShift 3.11 ships with a pre-configured and self-updating monitoring stack based on the Prometheus open source project. This ships with a set of alerts that notify the cluster admin of problems and a set of Grafana dashboards. The deployed Prometheus instance is responsible for monitoring and alerting on cluster and OpenShift components, it shouldn't be extended to monitor user applications. Users interested in leveraging Prometheus for application monitoring on OpenShift should consider using the operator lifecycle manager (https://github.com/operator-framework/operator-lifecycle-manager) to easily deploy a Prometheus Operator and setup new Prometheus instances to monitor and alert on their applications. Alertmanager is a cluster-global component for handling alerts generated by all Prometheus instances deployed in that cluster.
```

Comment:
Is this true, is there no application monitoring carried out at all by prometheus by default? For some reason I thought it would scrape metrics from applications too automatically as its discovers them.

Reply:
Yeah it does scrape metrics from pods - I think this means specifically changing it to monitor apps. I'll make the wording clearer.